sdk/src/main/java/co/omise/android/ui/OmiseListFragment.kt

```diff
@@ -106,6 +108,12 @@ class OmiseItemViewHolder(val view: View, val listener: OmiseListItemClickListen
             listItemView.alpha = 0.2F
         }
 
+        if (subtitleText.text != "") {
```

Comment:
```suggestion
        if (subtitleText.text.isNotBlank()) {
```

I think proper way to check empty string. https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/is-not-blank.html